test/rpmdbs/bdb-corrupt/var/lib/rpm/Packages

```diff
@@ -0,0 +1 @@
+CORRUPT
```

Comment:
I don't think any of the other files are necessary besides this one.

Reply:
Spot on. I thought I tried it and was unsuccessful, but I tried it again and it's working so I've removed the extras.